src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -447,6 +473,13 @@ EncodedResults StaticLLMPipeline::generate(
             m_kvcache_request.get_tensor(output_name).copy_to(kvcache_in_slice);
         }
     }
+    auto stop_time = std::chrono::steady_clock::now();
+    // If is called without tokenization then that stat will not be reported.
+    auto& metrics = results.perf_metrics;
+    metrics.num_input_tokens = 1 * input_ids.get_shape().at(1);
```

Comment:
I believe it's still should be `batch_size` instead of `1`. (Even though there is a limit that batch_size == 1)